horton/gbasis/test/test_ints.py

```diff
@@ -3035,12 +4488,13 @@ def test_electron_repulsion_water_ccpvdz_cart_hf():
 
 def check_g09_grid_fn(fn_fchk):
```

Comment:
Any function that does not start with `test_` should have docstrings. Can you add a docstring for one of the `check_` functions, e.g. this one? This could be as follows here:

    """"Integrate total density and compare with number of electrons.

    Parameters
    ----------
    fn_fchk : str
        The formatted checkpoint file containing a density matrix to be used for this test.
    """